examples/embedding/embedding.cpp

```diff
@@ -114,7 +116,9 @@ int main(int argc, char ** argv) {
     for (const auto & prompt : prompts) {
         auto inp = ::llama_tokenize(ctx, prompt, true, false);
         if (inp.size() > n_batch) {
-            inp.resize(n_batch);
+            fprintf(stderr, "%s: error: number of tokens in input line (%lld) exceeds batch size (%lld), increase batch size and re-run\n",
+                    __func__, (long long int) inp.size(), (long long int) n_batch);
```

Comment:
Here you can use `%ld` instead of `%lld`, no need to cast the type then

```suggestion
                    __func__, inp.size(), n_batch);
```

Reply:
applied & rolled back due to [build failure](https://github.com/ggerganov/llama.cpp/actions/runs/8429678709/job/23084305530).
IIRC, this is why I used `%lld` with casting in #6193. Although I tried your suggestion just in case 😉.

Reply:
We normally use `PRIu64 / PRId64` to print 64-bit integers. Alternatively, in this case just `%d` and cast to `(int)` is fine